src/api/schemas/activity-schema.ts

```diff
@@ -0,0 +1,31 @@
+import { any, number, object, string } from 'zod';
+
+const activitySchema = object({
+    // Required
+    title: string({
+        required_error: 'Title is required',
+        invalid_type_error: 'Title should be of type string'
+    }),
+    description: string({
+        required_error: 'Description is required',
+        invalid_type_error: 'Description should be of type string'
+    }),
+
```

Comment:
Are you missing some information to validate the request object? The businessId, the location (or address), etc.

*** see line 36 of `business-schema.ts`

Reply:
ah i forgot. i'll add them